packages/bundler-plugin-core/package.json

```diff
@@ -52,6 +52,8 @@
     "fix": "eslint ./src ./test --format stylish --fix"
   },
   "dependencies": {
+    "@babel/core": "7.18.5",
```

Comment:
Had to move this to a regular dependency, since it is necessary to run `transformAsync` from `@babel/core` during the transform hook